src/Component/Style/Style.tsx

```diff
@@ -175,6 +189,46 @@ export class Style extends React.Component<StyleProps, StyleState> {
     });
   }
 
+  onMultiEdit = (param: any) => {
+    switch (param.key) {
+      case 'color':
+        this.setState({colorModalVisible: true});
+        break;
+      case 'size':
+        this.setState({sizeModalVisible: true});
+        break;
+      case 'opacity':
+        this.setState({opacityModalVisible: true});
+        break;
+      default:
+    }
+  }
+
+  updateAllSelected = (value: any, property: string) => {
+    const style = _cloneDeep(this.state.style);
+    const selectedRules = style.rules.filter((rule: GsRule, index: number) => {
+      return this.state.selectedRowKeys.includes(index);
+    });
+    selectedRules.forEach((rule: GsRule) => {
+      rule.symbolizers.forEach((sym: GsSymbolizer) => {
+        sym[property] = value;
+      });
+    });
+    this.setState({style});
```

Comment:
You should also call `onStyleChange` here, if it exists.

Reply:
Changed in 8a6bb9b